tests/data/long_strings.py

```diff
@@ -207,6 +207,18 @@ def foo():
     " of it."
 )
 
+x = (
+    "........................................................................ \N{LAO KO LA}"
+)
+
+x = (
+    "........................................................................... \N{LAO KO LA}"
+)
+
+x = (
+    "............................................................................ \N{LAO KO LA}"
+)
+
```

Comment:
Maybe it would be appropriate to test the escaping too, that would help in testing the backslash algorithm.